crates/ra_ide/src/call_info.rs

```diff
@@ -1,22 +1,22 @@
 //! FIXME: write short doc here
 
-use ra_db::SourceDatabase;
+use hir::db::AstDatabase;
 use ra_syntax::{
-    algo::ancestors_at_offset,
     ast::{self, ArgListOwner},
-    match_ast, AstNode, SyntaxNode, TextUnit,
+    match_ast, AstNode, SyntaxNode,
 };
 use test_utils::tested_by;
 
-use crate::{db::RootDatabase, CallInfo, FilePosition, FunctionSignature};
+use crate::{db::RootDatabase, expand::descend_into_macros, CallInfo, FilePosition, FunctionSignature};
 
 /// Computes parameter information for the given call expression.
 pub(crate) fn call_info(db: &RootDatabase, position: FilePosition) -> Option<CallInfo> {
-    let parse = db.parse(position.file_id);
-    let syntax = parse.tree().syntax().clone();
+    let file = db.parse_or_expand(position.file_id.into())?;
+    let token = file.token_at_offset(position.offset).next()?;
+    let token = descend_into_macros(db, position.file_id, token);
 
     // Find the calling expression and it's NameRef
-    let calling_node = FnCallNode::with_node(&syntax, position.offset)?;
+    let calling_node = FnCallNode::with_node(&token.value.parent())?;
     let name_ref = calling_node.name_ref()?;
     let name_ref = hir::Source::new(position.file_id.into(), name_ref.syntax());
```

Comment:
@kjeremy After #2545 landed, if you change this line and rebase, the test will be passed. 

```suggestion
    let name_ref = token.with_value(name_ref.syntax());
```